sov-modules/sov-state/src/prover_storage.rs

```diff
@@ -59,6 +59,10 @@ impl<S: StorageSpec> Storage for ProverStorage<S> {
     type Witness = S::Witness;
     type RuntimeConfig = &'static str;
 
+    fn with_config(config: Self::RuntimeConfig) -> Result<Self, anyhow::Error> {
```

Comment:
Not sure when was the method introduced, but do we need it? It looks like it is not used anywhere.